Python/Product/Analysis/PythonAnalyzer.cs

```diff
@@ -784,9 +786,9 @@ class LazyModuleEnumerator {
             }
         }
 
-        #endregion
+#endregion
```

Comment:
Why does automatic formatting have to be so difficult... can we keep these indented? (Just the `#region` tags)